tests/mock_tests/portsorch_ut.cpp

```diff
@@ -455,6 +429,102 @@ namespace portsorch_test
         ts.clear();
     }
 
+    TEST_F(PortsOrchTest, LagMemberDoesNotCallSAIApiWhenPortIsAlreadyALagMember)
```

Comment:
i know all is writing this was but i would like to recommend adding a description of the motivation and the steps of each test.

Reply:
Done